src/Input/Input.js

```diff
@@ -176,6 +182,15 @@ export default class Input extends Component {
     }
   }
 
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
```

Comment:
Turns out I don't need this now, which is great. I'm trying to limit my changes to just the SelectField and SelectInputField.